src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml

```diff
@@ -1,11 +1,11 @@
 @model PlaceholderAdminNodeViewModel
 
-<style asp-src="~/OrchardCore.AdminTrees/Styles/admin-trees.css"at="Head" depends-on="admin"></style>
-<style asp-src="~/OrchardCore.AdminTrees/Styles/admin-trees-icon-picker.css" at="Head" depends-on="admin"></style>
+<style asp-src="~/OrchardCore.AdminMenu/Styles/admin-menu.min.css" debug-src="~/OrchardCore.AdminMenu/Styles/admin-menu.css" at="Head" depends-on="admin"></style>
```

Comment:
I see that the corresponding views on the Contents and the Lists modules are still referencing the old asset files: admin-trees.css , admin-trees-icon-picker.css, admin-trees.js, admin-trees-icon-picker.js

Reply:
done